api-test/src/main/resources/pms/OIDCClientEnhancement/DeactivateOIDCClient/DeactivateOIDCClient.yml

```
@@ -0,0 +1,35 @@
+DeactivateOIDCClientV2:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)